tools/c7n_mailer/c7n_mailer/aws/aws_smtp_delivery.py

```diff
@@ -0,0 +1,17 @@
+from c7n_mailer.smtp_delivery import SmtpDelivery
```

Comment:
Add headers

Reply:
Why does this class exist at all, we shouldnt be duplicating message transports across providers